.github/workflows/multichain-e2e-template.yml

```diff
@@ -0,0 +1,93 @@
+name: Multichain E2E Workflow Template
+
+on:
+  workflow_call:
+    inputs:
+      config:
+        required: true
+        type: string
+      relayer-type:
+        required: true
+        type: string
+
+jobs:
+  multichain-e2e:
+    runs-on: ubuntu-latest-16core
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: 'true'
+          path: ./agoric-sdk
+      - uses: ./agoric-sdk/.github/actions/restore-node
+        with:
+          node-version: 18.x
+          path: ./agoric-sdk
+
+      - name: yarn link
+        run: |
+          yarn link-cli ~/bin/agoric
+          echo "/home/runner/bin" >> $GITHUB_PATH
+        working-directory: ./agoric-sdk
+
+      - name: Enable Corepack
+        run: corepack enable
+        working-directory: ./agoric-sdk/multichain-testing
+
+      - name: Install @agoric/multichain-testing deps
+        run: yarn install
+        working-directory: ./agoric-sdk/multichain-testing
+
+      - name: Lint @agoric/multichain-testing
+        run: yarn lint
+        working-directory: ./agoric-sdk/multichain-testing
+
+      - name: Setup Starship Infrastructure
+        id: starship-infra
+        uses: cosmology-tech/starship-action@0.3.0
```

Comment:
why not latest? https://github.com/cosmology-tech/starship-action/releases

Reply:
Good question. I bumped to 0.5.3 [here](https://github.com/Agoric/agoric-sdk/pull/10216) and will see what CI thinks. 

It looks the ability to specify a helm chart version may have been lost starting 0.5.4 (the latest is 0.5.7). I filed an https://github.com/cosmology-tech/starship-action/issues/34 asking for clarification.

We can also bump the helm chart version. We're on 0.2.10 but it's up to 0.2.14. I ran into issues with the new `readinessProbe` - it seem the approach [we provided](https://github.com/cosmology-tech/starship/issues/545#issuecomment-2273551733) is not working on my local. I filed https://github.com/cosmology-tech/starship/issues/563 to track this

Reply:
563 was merged last week. Let's see if we can get an answer on 34 soon enough to incorporate it